src/lemlib/pid.cpp

```diff
@@ -93,6 +93,8 @@ float lemlib::FAPID::update(float target, float position, bool log) {
     prevOutput = output;
     prevError = error;
     totalError += error;
+
+    if (log) { Logger::logPid(name, output, kI * error, kD * totalError, kD * deltaError); }
```

Comment:
This line computes the outputs incorrectly
```suggestion
    if (log) { Logger::logPid(name, output, kP * error, kI * totalError, kD * deltaError); }
```